staging/operator-lifecycle-manager/deploy/chart/templates/0000_50_olm_01-networkpolicies.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
@anik120 I do not think it is needed to support Networkpolicy in layered operator.